apriltag/src/main/native/cpp/AprilTagFieldLayout.cpp

```diff
@@ -49,6 +50,11 @@ units::meter_t AprilTagFieldLayout::GetFieldWidth() const {
   return m_fieldWidth;
 }
 
+std::vector<AprilTag> AprilTagFieldLayout::GetTags() const {
+  auto vals = std::views::values(m_apriltags);
+  return std::vector<AprilTag>{vals.begin(), vals.end()};
```

Comment:
Could this return a `std::span` rather than a vector?

Reply:
I don't believe so as a span can only represent a block of contiguous memory and the values in the map are not guaranteed to be together in memory.

https://stackoverflow.com/questions/67509860/is-it-possible-to-construct-a-stdspan-from-a-view-in-c20

Reply:
No, as the underlying storage is a map.

Reply:
Most of our compilers don’t have views support yet, so this will need to be done more manually.

Reply:
GCC 12 and VS 2022 both have ranges support. Clang 15 has it behind `-fexperimental-library`, but we'd need Xcode 14 instead of 13.